exonum-light-client/src/main/java/com/exonum/client/ExonumHttpClient.java

```diff
@@ -15,26 +33,28 @@
 import okhttp3.RequestBody;
 import okhttp3.Response;
 
-class ExonumClientImpl implements ExonumClient {
-  private static final BaseEncoding HEX_ENCODER = BaseEncoding.base16().lowerCase();
+/**
+ * Implementation of the {@linkplain ExonumClient} which works over HTTP REST API.
+ * It uses {@linkplain OkHttpClient} internally for REST API communication with Exonum node.
+ */
```

Comment:
Shall we link https://exonum.com/doc/version/latest/advanced/node-management/ ? 
 I, for instance, often get lost in that structure.

Reply:
Didn't get what do you mean. Currently LC doesn't support node management

Reply:
I guess that page documents all the APIs of Exonum, including https://exonum.com/doc/version/latest/advanced/node-management/#transaction